config.md

```diff
@@ -29,7 +29,8 @@ For example, if a configuration is compliant with version 1.1 of this specificat
 **`root`** (object, REQUIRED) specifies the container's root filesystem.
 
 * **`path`** (string, OPTIONAL) Specifies the path to the root filesystem for the container. The path is either an absolute path or a relative path to the bundle.
-    Users SHOULD consider using a conventional name, such as `rootfs`.
+
+    The value SHOULD be the conventional `rootfs`.
```

Comment:
Neither the old nor the new wording applies to Windows (since #849, because `rootfs` is not a [GUID path][1]).  I [address that][2] in the in-flight #838, and am leaving it alone here since it seems orthogonal to dropping “consider”.  If folks do want me to address the “does not apply to Windows” issue in this PR as well, I'm happy to pull over some of the #838 wording to do that.

[1]: https://github.com/opencontainers/runtime-spec/blame/aaa24815e0ab632c424e18d726fd47f403ce2981/config.md#L34
[2]: https://github.com/opencontainers/runtime-spec/pull/838#issuecomment-304129858